scripts/exregional_run_prepstart.sh

```diff
@@ -94,6 +94,14 @@ YYYYMMDD=${YYYYMMDDHH:0:8}
 #
 #-----------------------------------------------------------------------
 #
+# Update SST currently set to run in the 01z cycle
+update_SST='01'
```

Comment:
Do we have a switch to turn off SST update?  If not, we might want to put "update_SST" into config.sh and then we can set like update_SST="-99" to skip SST update.

Reply:
It is hardcoded to run at 01Z, just as RAP/HRRR. For 09/14, 01Z didn't run at all. So yes it might be a good idea to set it configurable in config.sh, for two purposes: 1, setting it to something like "-99" to turn it off; 2, set it to a different time if needed. For example, when there is no 01Z run, then can we do it for a later valid time? @hu5970 @tanyasmirnova what do you think?

Reply:
I'd like to add an example here: RTMA will not do SST update at the moment. So it will be good to be able to turn on/off SST update from the config.sh file. Thanks!

Reply:
Changed "update_SST" to be configurable in config.sh and also added 
SST_ROOT="/lfs4/BMC/public/data/grids/ncep/sst/0p083deg/grib2"
update_SST="01"
to config.sh.RRFS_NA_3km as an example